cscs-checks/mch/collectives_halo.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, variant, bench_reference):
         }
 
         self.maintainers = ['AJ', 'MKr']
-        self.tags = {'production', 'mch', 'craype'}
+        self.tags = {'mch'}
```

Comment:
The tags should be removed only for Tsa/Arolla, right?

Reply:
Indeed, my bad.